### ISSUE TITLE
Doesn't build against latest OpenSSL version

### ISSUE THREAD

**Issue by ghost** [deleisha/evt-tls#9]
Running make, I get:

```
evt_tls.c:17:18: error: incomplete definition of type 'struct ssl_st'
    return t->ssl->server ? ENDPT_IS_SERVER : ENDPT_IS_CLIENT;
           ~~~~~~^

```
Building on mac, very latest versions of OpenSSL and libvu.

**Comment by deleisha** [comment]
Thanks for reporting this. Will look into this

**Comment by nuuman** [comment]
Will you look into this soon, or do I need to look for alternatives?

**Comment by deleisha** [comment]
Which version of openssl you are using. 

The API was introduced because openSSL < 1.0.2 does not have `SSL_is_server()`, which exactly does the same thing. But not sure why build fails. Are the openSSL header properly installed?

**Comment by jwinarske** [comment]
I have the same error building on Windows.

**Comment by deleisha** [comment]
Thanks jwinarske, Could you please do a PR, We will merge it

**Comment by deleisha** [comment]
Fixes with e12abc0. Closing this
